src/pages/SpacesDashboard/SpacesDashboard.tsx

```diff
@@ -98,100 +99,102 @@ export const SpacesDashboard: React.FC<SpacesDashboardProps> = ({
   const hasOtherVenues = renderedOtherVenues.length > 0;
 
   return (
-    <div className="SpacesDashboard">
-      <WithNavigationBar
-        variant="internal-scroll"
-        title={`${world?.name ?? ""}`}
-      >
-        <AdminRestricted>
-          <AdminTitleBar variant="grid-with-tools">
-            <ButtonNG
-              variant="secondary"
-              isLink
-              linkTo={ADMIN_IA_WORLD_BASE_URL}
-            >
-              Change world
-            </ButtonNG>
-            <AdminTitle>{world?.name} dashboard</AdminTitle>
-            <div>
-              {isWorldAdmin && (
+    <AdminLayout>
+      <div className="SpacesDashboard">
+        <WithNavigationBar
```

Comment:
[OOS] Something for next PR. Since there is now `<AdminLayout />` component, it basically does the same function as `<WithNavigationBar />` does.